eng/tools/lint-diff/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "target": "esnext",
```

Comment:
This is OK if you need it, but I'd like to keep our tools consistent if possible.  Should we upgrade all tools to this version?